src/libs/actions/Session/index.ts

```diff
@@ -331,6 +331,15 @@ function signInWithShortLivedAuthToken(email: string, authToken: string) {
                 isLoading: true,
             },
         },
+        // We are making a temporary modification to 'signedInWithShortLivedAuthToken' to ensure that 'App.openApp' will be called at least once
```

Comment:
Are we fine with `App.openApp` being called twice or more?

Reply:
ideally we don't want multiple API calls but in this instance we aren't adding anything that in itself will cause App.openApp to be called multiple times, but if that happens as a side effect from something else it shouldn't be a problem since we are just doing onyx merge